types/puppeteer/puppeteer-tests.ts

```diff
@@ -502,3 +502,20 @@ puppeteer.launch().then(async browser => {
         console.log(err instanceof TimeoutError);
     }
 });
+
+// domcontentloaded page event test
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  page.on('domcontentloaded', async () => {
+    page.evaluate(() => {
+      console.log('dom changed');
+    });
+  });
+
+  try {
```

Comment:
I think this code chunk is redundant for this test.

Reply:
@SimonSchick did you mean to remove `try-catch` and keep the `page.goto('.....` line?

Reply:
Yes, these tests are meant to test type safety, not functionality.